data/mods/Magiclysm/items/comestibles.json

```diff
@@ -18,6 +18,7 @@
     "stack_size": 1,
     "fun": -18,
     "flags": [ "FREEZERBURN" ],
+    "rot_spawn": "GROUP_EGG_OWLBEAR",
```

Comment:
what's the purpose of moving this line? i wouldn't change it

Reply:
If I'm understanding it right then having it on the egg yolk a) means that random eggs laid in the world won't hatch into more owlbears and b) a player could hatch a single egg into 40 owlbears.